src/Microsoft.AspNetCore.Razor.Language/Legacy/TagHelperDirectiveDescriptor.cs

```diff
@@ -1,48 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-
-namespace Microsoft.AspNetCore.Razor.Language.Legacy
-{
-    /// <summary>
-    /// Contains information needed to resolve <see cref="TagHelperDescriptor"/>s.
-    /// </summary>
-    internal class TagHelperDirectiveDescriptor
```

Comment:
This turned out to be only used in the TagHelperBinder which wasn't necessary anymore now that all the info is in the chunk generator.